integration-tests/src/test/java/org/apache/fineract/integrationtests/ClientTest.java

```diff
@@ -222,8 +222,9 @@ public void testClientAddressCreationWorks() {
         long postalCode = 1000L;
 
         // when
-        PostClientsAddressRequest addressRequest = new PostClientsAddressRequest().postalCode(postalCode).city(city).countryId(countryId)
-                .stateProvinceId(stateId).addressTypeId(addressTypeId.longValue()).isActive(addressIsActive);
+        PostClientsAddressRequest addressRequest = new PostClientsAddressRequest().postalCode(postalCode).city(city)
+                .countryId(Long.valueOf(countryId)).stateProvinceId(Long.valueOf(stateId)).addressTypeId(addressTypeId.longValue())
```

Comment:
We may also fix CodeHelper to return Long instead of Integer, then the above conversion would not be required

https://github.com/apache/fineract/blob/ff8a186b5cbd6756d0602ccc2622d131e9ff3d84/integration-tests/src/test/java/org/apache/fineract/integrationtests/ClientTest.java#L216-L219